app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -274,7 +274,7 @@ open class FenixApplication : LocaleAwareApplication() {
         return GlobalScope.async(Dispatchers.IO) {
             // ... but RustHttpConfig.setClient() and RustLog.enable() can be called later.
             RustHttpConfig.setClient(lazy { components.core.client })
-            RustLog.enable()
+            RustLog.enable(components.analytics.crashReporter)
```

Comment:
@grigoryk I just approved the AC PR and asked bors to land it. But can you confirm that those log messages are unlikely to contain PII?

Reply:
Error ones are definitely quite unlikely to contain PII. We already send them to Sentry in Firefox iOS, too. I'll double check with the a-s team, but I think we should be okay here.

Reply:
@eoger we're about to enable sending `error` log messages to Sentry. You don't expect them to have any PII, right?

Reply:
Yes [this should](https://github.com/mozilla/application-services/issues/2407) [be fine](https://github.com/mozilla/application-services/issues/909).